src/recovery/mod.rs

```diff
@@ -233,13 +254,71 @@ impl Recovery {
             self.hystart.start_round(pkt_num);
         }
 
+        // Pacing: Set the pacing rate if BBR is not used
+        if self.pacing_enabled &&
+            self.cc_algo == CongestionControlAlgorithm::CUBIC
```

Comment:
Is it cubic specific? I think it can be used with Reno as well.

Reply:
Added Reno also.